src/ProjectNeon/Assets/Battle/Enemies/EnemyVisualizer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyVisualizer : MonoBehaviour
+{
+
+    [SerializeField] private EnemyArea enemyArea;
+    [SerializeField] private GameObject enemy1;
+    [SerializeField] private GameObject enemy2;
+    [SerializeField] private GameObject enemy3;
+    [SerializeField] private GameObject enemy4;
+    [SerializeField] private GameObject enemy5;

```

Comment:
Can we make this a dynamically sized collection? 

We're not always going to have 5 enemies. Would be better if this algorithm could nicely space 1-7 enemies. 

For whatever part you can't do, at least open a ticket. Never use individual variables to represent an unknown collection size.